test/complexity_test.cc

```diff
@@ -88,7 +88,7 @@ const char *enum_big_o_1 = "\\([0-9]+\\)";
 // FIXME: Tolerate both '(1)' and 'lgN' as output when the complexity is auto
 // deduced.
 // See https://github.com/google/benchmark/issues/272
-const char *auto_big_o_1 = "(\\([0-9]+\\))|(lgN)";
+const char *auto_big_o_1 = "(\\([0-9]+\\))|(lgN)|(N\\^2)";
```

Comment:
Hm, i believe the name is still correct.
It is only used for the function where the true/correct complexity is `O(1)`,
we just fail to auto-detect it because of the noise in CI.

Reply:
`auto_big_o_1_or_lgn_or_n2_actually_we_have_no_idea_what_this_should_be`

Reply:
:)